src/client/common/variables/environmentVariablesProvider.ts

```diff
@@ -116,7 +116,8 @@ export class EnvironmentVariablesProvider implements IEnvironmentVariablesProvid
     }
     private getWorkspaceFolderUri(resource?: Uri): Uri | undefined {
         if (!resource) {
-            return;
+            const workspaceFolders = this.workspaceService.workspaceFolders || [];
```

Comment:
If we have just one workspace folder opened, then use that to load the env variables.